GeneralsMD/Code/Tools/MapCacheBuilder/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(MAPCACHEBUILDER_SRC
+    "Include/WinMain.h"
+    "Resource/Resource.h"
+    "Source/WinMain.cpp"
+)
+
+add_executable(mapcachebuilder WIN32)
+
+target_include_directories(mapcachebuilder PRIVATE
+    Include
+    Resource
+    ../../Libraries/Include
```

Comment:
Should we perhaps create aliases for these various include dirs in a higher up cmakelists? This `../../` is a bit unholy.

Reply:
Ideally it would link the target to get its includes but that needs a bunch of globals instantiating that gameengine requires. Maybe create an additional target called gameengineheaders that is just an interface to provide these?

Reply:
I have applied this improvement on my copy of your branch.